apps/prairielearn/src/lib/question-render.ts

```diff
@@ -574,11 +626,12 @@ export async function getAndRenderVariant(
     // On instructor question pages, only show if true answer is allowed for this question and there is at least one submission.
     resultLocals.showTrueAnswer = true;
   }
+  // if this isn't true, don't set `resultLocals.showTrueAnswer = false`
```

Comment:
Something super weird happened here. See c0a27fc92fcac936404c559d8b95a342afa3fb51

Reply:
Done in 8da92a754234aec4842a9b19c5df52b38aa13580